docs/mobile-apps/automated-testing/appium/real-devices.md

```diff
@@ -2,190 +2,209 @@
+## Configuring Appium Tests for Real Devices
 
-* Upload your app to an internal git repository, or private hosting solution with the necessary permissions (e.g. Amazon S3 with a strict bucket policy).
-* Ensure the hosted app URL is available to the machine running the automated test.
-* Ensure that you've enabled the **Require Sauce Connect/VPN** setting in your [organization's security settings](/basics/acct-team-mgmt/org-settings).
+Our [Test Configuration Options](/dev/test-configuration-options) reference documentation provides a complete index of required and optional parameters for Appium. Be aware that not all of the Appium capabilities list are supported for both virtual and real device tests.
 
-:::note
-Each session is a "fresh" installation of your app, meaning, you will not be able to access information about previous versions of your app.
-:::
+The following sections provide context and instructions for test configurations that are essential when using Appium to run automated tests on Sauce Labs **real devices**.
 
-## Uploading Mobile Apps with the Sauce Labs REST API
+### Specifying Your `app`
 
-Use the [App Storage REST API](/dev/api/storage/#upload-file-to-app-storage) to upload your app for Appium automated mobile testing. We currently support \*.apk or \*.aab Android app files and \*.ipa or \*.zip iOS app files (\*.zip files are parsed to determine whether a valid *.app bundle exists).
+In order to install your app on the devices you select, you must identify the location from which your mobile app can be accessed. You can specify a Sauce Labs Application Storage ID or filename, or a remote location to which Sauce Labs has access.
 
-:::caution Limited Support for *.aab Files
-At this time, \*.aab files are only supported for Android real device testing.
-:::
+```js title=App Storage Example
+caps.setCapability("app","storage:filename=mapp.zip");
+```
 
-### Application Storage and Data Center Endpoints
+```js title=Remote App Example
+caps.setCapability("app","https://github.com/test-apps/ios-app.zip");
+```
 
-When uploading your app for testing with real devices, you must identify the [Data Center](/basics/data-center-endpoints) where the devices you are testing are located by specifying the applicable API URL for that data center.
 
-Examples using the EU and US endpoints:
+### Setting the `appiumVersion`
 
-<Tabs
-  defaultValue="US Data Center"
-  values={[
-    {label: 'US Data Center', value: 'US Data Center'},
-    {label: 'EU Data Center', value: 'EU Data Center'},
-  ]}>
+If you omit the `appiumVersion` in your test configuration, your test runs with our default Appium version, which is typically the version that supports the broadest number of device combinations, but may not be the latest version. We recommend that you specify the newest Appium version that supports the specific devices and operating systems you plan to use for your test, so you can leverage the most up-to-date features and patches.
```

Comment:
Setting the `appiumVersion` is only recommended when people need a specific version, otherwise we advice not to set it. the downside of setting it to a specific version is that they test a long time agains an old version. 

Can you add that in the text instead of the above?

Reply:
I actually decided to remove this section from this page, since these configs are not comprehensive -- they are supposed to represent the most common config use cases. I did add some additional details to the Test Config Options page in the `appiumVersion` entry, which I think is a better place for this information.